src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -380,6 +380,11 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'default': false,
 				'description': localize('perEditorGroup', "Controls if the limit of maximum opened editors should apply per editor group or across all editor groups.")
 			},
+			'workbench.editor.showProblems': {
```

Comment:
I wonder why we need this, can we not expose the existing `renderValidationDecorations` from the editor as a editor setting?

Reply:
That's a good point actually. I think atm, it is in `EditorOptions`, and this means we wouldn't have to use check for the setting contribution everywhere I think?

Similar to https://github.com/justschen/vscode/blob/3a9916c55a4bd93d733d88ca9ce84ab1249c5fc1/src/vs/editor/common/viewModel/viewModelImpl.ts#L260

Reply:
Yeah it should just work like other editor options that apply directly. I would also think this enables the feature for all text based editors: diff, merge, notebook cells.

Reply:
No longer using a new setting, but settings UI can control `renderValidationDecorations`